bindings/redis/redis.go

```diff
@@ -48,32 +45,21 @@ func NewRedis(logger logger.Logger) *Redis {
 
 // Init performs metadata parsing and connection creation
 func (r *Redis) Init(meta bindings.Metadata) error {
-	m, err := r.parseMetadata(meta)
+	_, err := r.parseMetadata(meta)
```

Comment:
Why is this parsing still needed if the output is dropped?

Reply:
Because of there are two config `maxRetries` and `maxRetryBackoff` are parsed but not used. And I think just delete these code may not be a good idea, at least it shouldn’t be in this pr. Maybe a separate issue should be opened to discuss this.